src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -41,12 +42,12 @@ private JsonEncodedText(byte[] utf8Value)
         /// <exception cref="ArgumentException">
         /// Thrown when the specified value is too large or if it contains invalid UTF-16 characters.
         /// </exception>
-        public static JsonEncodedText Encode(string value)
+        public static JsonEncodedText Encode(string value, JavascriptEncoder encoder = null)
```

Comment:
casing is wrong, same goes for all the places where you added this: `JavascriptEncoder` -> `JavaScriptEncoder`

Also, you will get build errors because you have xml docs and haven't added the encoder param into the docs, like: `/// <param name="encoder"></param>`

Reply:
Lol, yep! Nice catch. I appreciate a second set of eyes to catch mistakes like these.